jobs/aks-cluster-only.yaml

```diff
@@ -21,4 +24,11 @@ jobs:
       aksResourceGroup: ${{ parameters.aksResourceGroup }}
       aksParametersFile: ${{ parameters.aksParametersFile }}
       aksServicePrincipalId: $(aks-sp-id)
-      aksServicePrincipalSecret: $(aks-sp-secret)
\ No newline at end of file
+      aksServicePrincipalSecret: $(aks-sp-secret)
+  - template: ../tasks/enable-aks-autoscaler.yaml
```

Comment:
is something making this conditional? or have you just pushed it to test on the test cluster?

Reply:
just pushed to test